### PR TITLE
Map Select: Docs and Cleanup

### PR DESCRIPTION
**Pull request by engineer124**
Cleaning up 1 or 2 things turned into cleaning up an entire file.

### CODE REVIEW

**Review by jpburnett — APPROVED** [review]

File: src/overlays/gamestates/ovl_select/z_select.c [hunk 1]
```diff
@@ -620,62 +764,58 @@ void Select_PrintMenu(SelectContext* this, GfxPrint* printer) {
...
-    // "Please wait just a minute",
-    "\x8Dｲﾏｼﾊﾞﾗｸｵﾏﾁｸﾀﾞｻｲ",
-    // "Don't worry, don't worry. Take a break, take a break",
-    "\x8Dｱﾜﾃﾅｲｱﾜﾃﾅｲ｡ﾋﾄﾔｽﾐﾋﾄﾔｽﾐ｡",
+    // "Please wait a minute"
+    GFXP_HIRAGANA "ｼﾊﾞﾗｸｵﾏﾁｸﾀﾞｻｲ",
+    // "Hold on a sec"
+    GFXP_HIRAGANA "ﾁｮｯﾄ ﾏｯﾃﾈ",
+    // "Wait a moment"
+    GFXP_KATAKANA "ｳｪｲﾄ ｱ ﾓｰﾒﾝﾄ",
```
Comment (isghj5) [inline src/overlays/gamestates/ovl_select/z_select.c:773]: Man I heard you the first time

File: tools/disasm/variables.txt [hunk 2]
```diff
@@ -970,7 +970,7 @@
     0x801BDBA0:("D_801BDBA0","UNK_TYPE1","",0x1),
     0x801BDBA4:("D_801BDBA4","UNK_TYPE1","",0x1),
     0x801BDBA8:("D_801BDBA8","UNK_TYPE1","",0x1),
-    0x801BDBB0:("D_801BDBB0","u8","",0x1),
+    0x801BDBB0:("gWeatherMode","u8","",0x1),
```
Comment (isghj5) [inline tools/disasm/variables.txt:973]: Should this change be added to namefixer or was it only accessed by code?

Comment (engineer124) [reply tools/disasm/variables.txt:973]: I just did a double check, and it seems like the only undecompiled file left that uses it is `z_kankyo`. I checked roz's github and it doesn't look like that file has been started yet, so it should be fine to not add it?

Comment (isghj5) [reply tools/disasm/variables.txt:973]: Probably fine then yeah

**Review by isghj5 — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**